### PR TITLE
update notebook-pr.yaml

### PR DESCRIPTION
**Pull request by spirosChv**
https://github.com/NeuromatchAcademy/course-content-dl/blob/workflow_update/.github/workflows/notebook-pr.yaml#L64-L76

### CODE REVIEW

**Review by mwaskom — APPROVED** [review]
This code looks right to me. I would err on the safe side and test the other branch of the conditional (i.e. push a notebook without the ci:execute flag and make sure that process notebooks just checks execution). But I don't want to hold this up so merge when you're satistfied.

File: .github/workflows/notebook-pr.yaml [hunk 1]
```diff
@@ -47,7 +47,7 @@ jobs:
           echo ci/ >> .gitignore
 
       - name: Install dependencies
-        if: "!contains(env.COMMIT_MESSAGE, 'skip ci')"
+        if: "contains(env.COMMIT_MESSAGE, 'ci:execute')"
```
Comment (mwaskom) [inline .github/workflows/notebook-pr.yaml:50]: I think you probably want both conditions here, right?

Comment (mwaskom) [reply .github/workflows/notebook-pr.yaml:50]: i.e., install the dependencies if you are (a) not skipping ci AND (b) executing the notebooks

Comment (spirosChv) [reply .github/workflows/notebook-pr.yaml:50]: correct I m fixing it

**Comment by mwaskom** [comment]
I guess one remaining question is what you think the default behavior should be. If the majority of DL notebooks will be too slow to feasibly execute during CI, it probably makes sense to keep the default as non-execution. But if most notebooks will be fine and just a few notebooks will be too slow, the execution check is more rigorous and so it might make sense to change to logic here to have that be the default. (i.e. execute unless `ci:noexecute` or similar appears in the commit msg).

**Comment by spirosChv** [comment]
> I guess one remaining question is what you think the default behavior should be. If the majority of DL notebooks will be too slow to feasibly execute during CI, it probably makes sense to keep the default as non-execution. But if most notebooks will be fine and just a few notebooks will be too slow, the execution check is more rigorous and so it might make sense to change to logic here to have that be the default. (i.e. execute unless `ci:noexecute` or similar appears in the commit msg).

I will see in the next days how it goes with the other notebooks and I will decide. Bute you are right, makes sense

**Pull request merged**